corehq/apps/linked_domain/views.py

```diff
@@ -357,6 +357,9 @@ def delete_domain_link(self, in_data):
     def create_release(self, in_data):
         push_models.delay(self.domain, in_data['models'], in_data['linked_domains'],
                           in_data['build_apps'], self.request.couch_user.username)
+
+        track_workflow(self.request.couch_user.username, f"Linked domain: pushed data models {in_data['models']}")
```

Comment:
It's more typical to pass the dynamic data using the `properties` kwarg of `track_workflow`.